app/assets/apple-pay/test-12/apple-developer-merchantid-domain-association.txt

```diff
@@ -1,72 +1,72 @@
+bDALBgNVHQ8EBAMCB4AwEQYLKoZIhvdjZAYCAgEEAgUAMA0GCSqGSIb3DQEBBQUAA4IBAQCKtlZO
+QdKx8yFDnQvZx_-0mFiZ7UGnMPHv29qAbSG2s1zcvofxOSG8xKHaJtLtpvHtwBGVGaxHuCoOaZjU
+5ubOa-oiwSunAW5BTCwd4yOMhiejjBeczBvu2GQzD0QOiztIYhNiVNNxTZItgTr_psuOXENxbIgz
+IjwgPx14uf8aKiHoc2nPm6Gh4T3pDBe8qXPX6VK6wrYfoSKFPrhcGrKae5_My1A6_pbH_EhH9CSi
+rT5M-0rbZ5wDcRUnP4NS-7L0GRapPqnsFOxewgqn803RMKrUrbgyoVAvTVXIQiesHM5yZPAs4Ank
+TLkLIDU3nNULkcQxHdAr4lyCvX3Q_giZMIIEuzCCA6OgAwIBAgIBAjANBgkqhkiG9w0BAQUFADBi
+MQswCQYDVQQGEwJVUzETMBEGA1UEChMKQXBwbGUgSW5jLjEmMCQGA1UECxMdQXBwbGUgQ2VydGlm
+aWNhdGlvbiBBdXRob3JpdHkxFjAUBgNVBAMTDUFwcGxlIFJvb3QgQ0EwHhcNMDYwNDI1MjE0MDM2
+WhcNMzUwMjA5MjE0MDM2WjBiMQswCQYDVQQGEwJVUzETMBEGA1UEChMKQXBwbGUgSW5jLjEmMCQG
+A1UECxMdQXBwbGUgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkxFjAUBgNVBAMTDUFwcGxlIFJvb3Qg
+Q0EwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDkkakJH5HbHkdQ6wXtXnmELes2oldM
+VeyLGYne-Uts9QerIjAC6Bg--FAJ039BqJj50cpmnCRrEdCju-QbKsMflZ56DKRHi1vUFjczy8QP
+Tc4UadHJGXL1XQ7Vf1-b8iUDulWPTV0N8WQ1IxVLFVkds5T39pyez1C6wVhQZ48ItCD3y6wsIG9w
+tj8BMIy3Q88PnT3zK0koGsj-zrW5DtleHNbLPbU6rfQPDgCSC7EhFi501TwN22IWq6NxkkdTVcGv
+L0Gz-PvjcM3mo0xFfh9Ma1CWQYnEdGILEINBhzOKgbEwWOxaBDKMaLOPHd5lc_9nXmW8Sdh2nzMU
+ZaF3lMktAgMBAAGjggF6MIIBdjAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH_BAUwAwEB_zAdBgNV
+HQ4EFgQUK9BpR5R2Cf70a40uQKb3R01_CF4wHwYDVR0jBBgwFoAUK9BpR5R2Cf70a40uQKb3R01_
+CF4wggERBgNVHSAEggEIMIIBBDCCAQAGCSqGSIb3Y2QFATCB8jAqBggrBgEFBQcCARYeaHR0cHM6
+Ly93d3cuYXBwbGUuY29tL2FwcGxlY2EvMIHDBggrBgEFBQcCAjCBthqBs1JlbGlhbmNlIG9uIHRo
+aXMgY2VydGlmaWNhdGUgYnkgYW55IHBhcnR5IGFzc3VtZXMgYWNjZXB0YW5jZSBvZiB0aGUgdGhl
+biBhcHBsaWNhYmxlIHN0YW5kYXJkIHRlcm1zIGFuZCBjb25kaXRpb25zIG9mIHVzZSwgY2VydGlm
+aWNhdGUgcG9saWN5IGFuZCBjZXJ0aWZpY2F0aW9uIHByYWN0aWNlIHN0YXRlbWVudHMuMA0GCSqG
+SIb3DQEBBQUAA4IBAQBcNplMLXi37Yyb3PN3m_J20ncwT8EfhYOFG5k9RzfyqZtAjizUsZAS2L70
+c5vu0mQPy3lPNNiiPvl4_2vIB-x9OYOLUyDTOMSxv5pPCmv_K_xZpwUJfBdAVhEedNO3iyM7R6PV
+byTi69G3cN8PReEnyvFteO3ntRcXqNx-IjXKJdXZD9Zr1KIkIxH3oayPc4FgxhtbCS-SsvhESPBg
+OJ4V9T0mZyCKM2r3DYLP3uujL_lTaltkwGMzd_c6ByxW69oPIQ7aunMZT7XZNn_Bh1XZp5m5MkL7
+2NVxnn6hUrcbvZNCJBIqxw8dtk2cXmPIS4AXUKqK1drk_NAJBzewdXUhMYICjDCCAogCAQEwgYUw
+eTELMAkGA1UEBhMCVVMxEzARBgNVBAoTCkFwcGxlIEluYy4xJjAkBgNVBAsTHUFwcGxlIENlcnRp
+ZmljYXRpb24gQXV0aG9yaXR5MS0wKwYDVQQDEyRBcHBsZSBpUGhvbmUgQ2VydGlmaWNhdGlvbiBB
+dXRob3JpdHkCCD1yIOPPjPIlMAkGBSsOAwIaBQCggdwwGAYJKoZIhvcNAQkDMQsGCSqGSIb3DQEH
+ATAcBgkqhkiG9w0BCQUxDxcNMjAwMjAzMTE1NjQ1WjAjBgkqhkiG9w0BCQQxFgQUeZa8p8PhNNHE
+gbQ4P3K8gQCftxUwKQYJKoZIhvcNAQk0MRwwGjAJBgUrDgMCGgUAoQ0GCSqGSIb3DQEBAQUAMFIG
+CSqGSIb3DQEJDzFFMEMwCgYIKoZIhvcNAwcwDgYIKoZIhvcNAwICAgCAMA0GCCqGSIb3DQMCAgFA
+MAcGBSsOAwIHMA0GCCqGSIb3DQMCAgEoMA0GCSqGSIb3DQEBAQUABIIBAHTh_C1wzInT3muqdMrb
+Hr6clXUR5ZVpf2UE1N1t0xJ-_9Qv9A9OanLVAgtrDO1GuwyNPqdhS47yIFL6w8MF1Z3-4LC9YAm8
+_l9rUkxEYiJa50TH8okH7pP6AQT1q_jl18lpsz6NiczWuGePDzKnwzlLm3CAmAFzpbRsYo_aUCDD
+5hJYDBfS_yQyScQOSJPmet6sEP80bWSkqf71QNsnVo7Fqw2CUY8O9d8f4mWe9l0o9Ko_2ynLaFj5
+1eR3zwnzgvRuSMsK2TWk8kMOLz87WGq721pFFLPV__e4F3UFK_MJoIuvKJtv8IIyZDEGWSJz_Ekl
+iajV91M9KSswwZgs6Wo
```

Comment:
Curious as to why this file has changed. We already re-verified test-12. Did we download the file again afterwards?

Reply:
I'm not sure how that actually changed, we did have a merge conflict. May be when resolved we picked up an old version. Let us fix that so it is the version you checked in last Thursday.

Reply:
I have just checked the contents of stage/prod they are correct. Just this one need to be changed

Reply:
We have now corrected the test file.